README.md

```diff
@@ -27,15 +26,27 @@ contributor guidelines.
 
 ### Run website locally
 
-Refer to this [doc](https://github.com/knative/website/blob/main/DEVELOPMENT.md) in the website repo.
+For instructions, see Knative's [MkDocs contributor guide](https://knative.dev/docs/help/contributor/mkdocs-contributor-guide).
 
 ### Website source files
 
 Source files for the documentation on the website are located within the
 [`/docs`](docs) directory of this repo.
 
+### Documentation versions for Knative releases
+
+Each release of the Knative docs is available in the website (starting with
+0.3) and their source files are all stored in
+[branches of this repo](./doc-releases.md).
+
+## Contributing to docs
+
+We're excited that you're interested in contributing to the Knative documentation! Check out the resources below to get started.
+
+### Getting started
+
 If you want to contribute a fix or add new content to the documentation, you can
-navigate through this repo or use the `Edit this page` on each of the pages of
+navigate through the [`/docs`](./docs) repo or use the `Edit this page` pencil icon on each of the pages of
```

Comment:
```suggestion
navigate through the [`/docs`](docs/index.md) repo or use the `Edit this page` pencil icon on each of the pages of
```

Reply:
I don't think the file `index.md` exists. Also, the aim of this is to link to a folder in GitHub so that user know they need to go the the `docs` folder in the docs repo. It is not for the website.

Reply:
Okay. but the link should nonetheless be `[`/docs`](docs)` then, right? We should at least cut off the `./`.

Reply:
OK